docs/documentation/index.html

```diff
@@ -198,9 +238,9 @@ <h4>Source Group setup for C/C++</h4>
 						</div>
 					</div>
 
-					<strong>Create empty</strong>
+					<strong>Create Empty</strong>
 
-					<p>If none of the above options apply to your project, please continue at <a href="#CreateAnEmptyCXXProject">Create an empty C/C++ Source Group</a> and come back here as soon as the project is created.</p>
+					<p>If either of the above options apply to your project, please continue at <a href="#CreateAnEmptyCXXProject">Create an empty C/C++ Source Group</a> and come back here as soon as the project is created.</p>
```

Comment:
Wouldn't it be "neither"? What we want to say here is that the "Empty Source Group" is the fallback approach if nothing else is applicable.